src/action/grpc.js

```diff
@@ -49,6 +49,31 @@ class GrpcAction {
     return this._sendIpc('unlockRequest', 'unlockResponse', method, body);
   }
 
+  //
+  // Autopilot grpc client
+  //
+
+  /**
+   * This is called to initialize the GRPC client to autopilot. Once `autopilotReady`
+   * is set to true on the store GRPC calls can be made to the client.
+   * @return {Promise<undefined>}
+   */
+  async initAutopilot() {
+    await this._sendIpc('lndAtplInit', 'lndAtplReady');
+    this._store.autopilotReady = true;
+    log.info('GRPC autopilotReady');
+  }
+
+  /**
+   * Wrapper function to execute calls to the autopilot grpc client.
+   * @param  {string} method The autopilot GRPC api to call
+   * @param  {Object} body   The payload passed to the api
+   * @return {Promise<Object>}
+   */
+  async sendAutopilotCommand(method, body) {
+    return this._sendIpc('lndAtplRequest', 'lndAtplResponse', method, body);
+  }
+
```

Comment:
@valentinewallace I also moved these back to the grpc module since there was quite a bit of copy/paste ;)